Extensions/AboutUsParser.js

```diff
@@ -162,11 +173,11 @@ function writeFile(path, string, appendFile) {
     }
 }
 
-/** Creates a Json file with our data
+/** Creates a Json file with our data stringified as Json
  * 
- * @param {Array} array Array of `new Paragraph()` objects
+ * @param {String} string string to be stringified and saved as a json file
  */
-function saveAsJson(array) {
+function saveAsJson(string) {
```

Comment:
wait, this is broken - it's called string but in the function you still use array. I think best would be to just call it "object" or something like that - as the input can be anything that is stringifiable (of which most general version is simply object)